Libraries/Drei/views.lisp

```diff
@@ -73,7 +73,10 @@ If empty, tabs every TAB-WIDTH are assumed.")))
                    (recorded-stream recorded-stream)) tabify
     (unless (eq stream recorded-stream)
       ;; Update the recorded values.
-      (setf space-width (stream-character-width stream #\Space)
```

Comment:
please use `text-style-character-width` for that (also in climb package).

Reply:
done